<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1474</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Учет ЗИП</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>830</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ВЫХОД</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текущая дата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текущее время</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>160</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>1183</x>
      <y>600</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1141</width>
      <height>861</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Запасные части</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ЗИП к покупным</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Крепежные детали</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tableView_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Инструмент</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tableView_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Принадлежности</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableView" name="tableView_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Материалы</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableView" name="tableView_6"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioBut_all">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>680</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ВСЕ</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBut_shortage">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>680</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>НЕКОМПЛЕКТ</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>660</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>ОТОБРАЖЕНИЕ ЭЛЕМЕНТОВ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_red">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>750</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Разрешить редактирование наличия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>570</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>exit_button</zorder>
   <zorder>lineEdit_date</zorder>
   <zorder>lineEdit_time</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>lineEdit_search</zorder>
   <zorder>tabWidget1</zorder>
   <zorder>radioBut_all</zorder>
   <zorder>radioBut_shortage</zorder>
   <zorder>checkBox_red</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1474</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
